src/proof/nova.rs

```diff
@@ -65,22 +65,22 @@ impl<'a> MultiFrame<'a, S1, IO<S1>, Witness<S1>> {
 }
 
 pub struct NovaProver<F: LurkField> {
-    chunk_frame_count: usize,
+    reduction_count: usize,
```

Comment:
A one-liner comment here saying that each Nova step will carry over `reduction_count` steps of the small-step abstract amchine of Lurk would help.